clang/test/dpct/asm/prefetch_eviction_priority.cu

```diff
@@ -0,0 +1,28 @@
+// UNSUPPORTED: cuda-8.0, cuda-9.0, cuda-9.1, cuda-9.2, cuda-10.0, cuda-10.1, cuda-10.2, cuda-11.0, cuda-11.1, cuda-11.2, cuda-11.3
```

Comment:
This lit test could be removed; For the not-supported scenarios, the current ASM migration framework will emit a warning message for the "SYCLGenError()" scenario.